app-backend/db/src/main/scala/AnnotationLabelDao.scala

```diff
@@ -269,4 +269,14 @@ object AnnotationLabelDao extends Dao[AnnotationLabelWithClasses] {
       )
       """.update.run
     } yield ()
+
+  def toggleBySessionIds(
+      sessionIds: NonEmptyList[UUID],
+      setActive: Boolean
+  ): ConnectionIO[Int] = {
+    (fr"UPDATE " ++ tableF ++ fr"""SET
+      is_active = ${setActive}""" ++ Fragments.whereAndOpt(
```

Comment:
I think, if this is a toggle, we can get rid of the `setActive` field. For example, we could instead in postgres do:

`SET is_active = not is_active`

and not worry about setting the statuses separately in the completion endpoint, just traverse the whole list of session ids and flip the switches.

This also makes testing a little easier, since calling the toggle function twice should get you back whatever active labels you started with.